themes-default/slim/views/schedule.mako

```diff
@@ -19,6 +19,82 @@ const startVue = () => {
         el: '#vue-wrap',
         data() {
             return {};
+        },
+        mounted() {
+            if ($.isMeta({ layout: 'schedule' }, ['list'])) {
+                const sortCodes = {
+                    date: 0,
```

Comment:
Switching sorting breaks the page.

![image](https://user-images.githubusercontent.com/1331394/37874208-3781c0f0-302b-11e8-94f0-5db1ffc0e959.png)

Reply:
I'll fix this.

Reply:
This should be fixed now